lua/entities/gmod_tardis/modules/sh_player.lua

```diff
@@ -0,0 +1,15 @@
+if SERVER then
+	ENT:AddHook("StopMat", "EnterOnTP", function(self)
+		local min, max = self:GetCollisionBounds()
+		min = self:LocalToWorld(min)
+		max = self:LocalToWorld(max)
+		local entities = ents.FindInBox(min, max)
+		if #entities != 0 then
+			for k,v in pairs(entities) do
+				if v:IsPlayer() then
+					self:PlayerEnter(v)
```

Comment:
How are we going to protect against potential abuse here? I think there should be a global on/off switch at the server, maybe worth thinking about other kinds of protection too but it needs this at minimum